### PR TITLE
USWDS-Site - About: Update vision, mission, polestar

### LINKED ISSUE #2336: Update mission, vision, and polestar on USWDS website
### Purpose
To update the mission, vision, and polestar on the USWDS website.

### Definition of done

[About page](https://designsystem.digital.gov/about/) has been updated to reflect USWDS's new mission, vision, and polestar.

> **Mission**
> Shaping the future of government digital services.
> 
> **Vision**
> Empowered and supported digital service teams. Familiar and easy-to-use digital services.
> 
> **Polestar**
> We help government teams align, design, and keep their websites and services up to date.

### Will be closed by
 PR [2397](https://github.com/uswds/uswds-site/pull/2397)

### PR DESCRIPTION
**Pull request by finekatie**
Updating Mission, Vision and Polestar.

# Summary

To update the mission, vision, and polestar on the USWDS website. [About page](https://designsystem.digital.gov/about/) has been updated to reflect USWDS's new mission, vision, and polestar.

## Related issue

Closes #2336 

## Preview link

Preview link: [About page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/kf-mission-vision-updates/about/)

## Problem statement

Mission, vision and polestar were recently revisited and revised, but not on the site. This update will update the new mission, vision and polestar on the site.

## Solution

This new copy will update the mission, vision and polestar on the site to reflect what's current. Simple copy changes made here.  

The vision, mission, and polestar text were pulled from the [September MC script](https://digital.gov/event/2023/09/21/uswds-monthly-call-september-2023/).

## Major changes

n/a

## Testing and review

Request dev review and fed review to ensure copy updates are correct and page structure remains intact.

### CODE REVIEW

**Comment by finekatie** [comment]
Hi @amyleadem and @mahoneycm , Sarah and I updated this page - hoping you can take a look before we request final approval from Anne and Dan. Thank you!

**Review by mahoneycm — APPROVED** [review]
Approved 👍

**Comment by finekatie** [comment]
Thought I had assigned to @annepetersen  and @thisisdano yesterday to review, but looks like I did not. Have updated again to add them for review and approval.

**Comment by finekatie** [comment]
@annepetersen made changes on 2/7 and noted in issue. This still needs review for approval and merge. :)

**Comment by annepetersen** [comment]
The thing I'm hung up on is that the last "[where we’ve been and where we’re going](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/kf-mission-vision-updates/about/[https://youtu.be/rl5hgoOK22o])" link does not work currently, as far as I can tell.

**Comment by finekatie** [comment]
I see, @annepetersen  - oof! Here's the original link: https://designsystem.digital.gov/whats-new/updates/2017/12/20/2017-where-weve-been-where-were-going/
But it's from 2017. Is that still relevant?

**Comment by annepetersen** [comment]
[The revision I made here](https://github.com/uswds/uswds-site/blob/kf-mission-vision-updates/pages/about/overview.md) just omits it entirely.

**Review by thisisdano — APPROVED** [review]

File: pages/about/website-policies-and-notices.md
Comment (thisisdano) [inline pages/about/website-policies-and-notices.md:185]: Thanks for fixing this 404 in this PR

**Review by annepetersen — APPROVED** [review]
LGTM!

**Comment by thisisdano** [comment]
Merging over pa11y crash.

**Pull request merged**
